services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -1978,6 +2000,29 @@ public Builder unrecognizedProperties(@Nullable Map<String, String> unrecognized
       return unrecognized(null);
     }
 
+    /***
+     * Enables filtering the route and lane by payment methods. Default is no filtering.
+     * Available payment methods match {@link DirectionsCriteria.PaymentMethodsCriteria}
+     * @param paymentMethods is a comma separated string where each value
+     *                       matches {@link DirectionsCriteria.PaymentMethodsCriteria}
+     * @return this builder
+     */
+    @NonNull
+    public abstract Builder paymentMethods(@Nullable String paymentMethods);
```

Comment:
We're missing updates to `MapboxDirections` that use this new parameter (and corresponding request/response tests).

Reply:
Added. But I'm not sure about response tests in the `services-directions` module. They seem like a duplication of tests from the `services-dirctions-model` module, as they just check how a JSON test file transforms to model. WDYT?